scripts/report_scripts/index.Rmd

```diff
@@ -22,6 +22,18 @@ library(reshape2)
 df_var <- read.table(var_timecourse_csv, sep = "\t", header=T)
 df_mut <- read.table(mut_timecourse_csv, sep = "\t", header=T)
 ```
+PiGx ww sars-cov-2 performs mutation analysis of SARS-CoV-2 and reports and quantifies the occurrence of 
```

Comment:
I agree. 
But also on that, we need a name. PiGx sars-cov-2-ww is weird and also not pronounceable. 
Maybe we need something more abstract here... SSC2 (sewage instead of wastewater) or smth?

Reply:
Altuna is the mother of names.  Better let him incubate a final name.